frontend/src/components/account/my-account/Account.js

```diff
@@ -33,7 +33,7 @@ class Account extends Component {
       allergies: [],
       isLoading: true,
       error: null,
-      signOut: signOut !== null ? signOut: this.fetchSignOut()
+      signOut: signOut !== null ? signOut : this.fetchSignOut(),
```

Comment:
Sorry I just saw this, but I'd suggest replacing `signOut` with `signedOut` and in other files `isSignIn` for `signedIn` (no need for "is" since it's a boolean)

Reply:
In this case, signOut is a url, so would signOutUrl be more clear?